src/providers/twitch/TwitchBadges.cpp

```diff
@@ -7,60 +7,95 @@
 
 #include "common/NetworkRequest.hpp"
 #include "common/Outcome.hpp"
+#include "common/QLogging.hpp"
 #include "messages/Emote.hpp"
```

Comment:
A bit unrelated, but I believe that Qt-specific includes can be moved under our includes in quotation marks.

Reply:
We don't have a standard of include order, so I'll mark this resolved for now.